group_vars/matrix_servers

```diff
@@ -4466,6 +4469,7 @@ matrix_synapse_reverse_proxy_companion_container_labels_traefik_hostname: "{{ ma
 
 matrix_synapse_reverse_proxy_companion_container_labels_public_client_synapse_client_api_enabled: "{{ matrix_synapse_container_labels_public_client_synapse_client_api_enabled }}"
 matrix_synapse_reverse_proxy_companion_container_labels_public_client_synapse_admin_api_enabled: "{{ matrix_synapse_container_labels_public_client_synapse_admin_api_enabled }}"
+matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_enabled: "{{ matrix_synapse_container_labels_internal_client_synapse_admin_api_enabled }}"
```

Comment:
Like `matrix_synapse_container_labels_internal_client_synapse_admin_api_enabled`, the `matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_enabled` variable is written to here, but it's not defined in `defaults/main.yml` in the `matrix-synapse-reverse-proxy-companion` role.

There should be a definition with some default value (`false`) there.